app/mailers/candidate_mailer.rb

```diff
@@ -124,6 +124,48 @@ def application_rejected_offers_only(application_choice)
     )
   end
 
+  def application_withdrawn_on_request_all_applications_withdrawn(application_choice)
+    @course = application_choice.course_option.course
```

Comment:
We have a `course` through `course_option` association in place so this can be simplified.

```suggestion
    @course = application_choice.course
```

Reply:
Yes I get a lot of test failures when using the `through` association as `@course` is `nil` presumably in some cases this is to do with how the tests are set up but also the mailer preview specs fail.

Reply:
👍🏼